src/main/java/com/inversoft/rest/RESTClient.java

```diff
@@ -351,6 +351,20 @@ public URI getURI() {
     return this;
   }
 
+  /**
+   * Add URL parameters from a {@code Map<String, Object>}.
+   *
+   * @param parameters The url parameters <code>Map</code> to add.  For each item in the <code>Map</code>
+   *                   this will call <code>urlParameter(String, Object)</code>
+   * @return This.
+   */
+  public RESTClient<RS, ERS> urlParameter(Map<String, Object> parameters) {
```

Comment:
I would vote to name this `urlParameters` (plural)

Reply:
Yeah. That's the Java convention, which is different than other languages. Multiples using `s` to be grammatically aligned.

Reply:
Done.